lib/salomon_bottom_bar.dart

```diff
@@ -56,103 +60,106 @@ class SalomonBottomBar extends StatelessWidget {
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
 
-    return SafeArea(
-      minimum: margin,
-      child: Row(
-        /// Using a different alignment when there are 2 items or less
-        /// so it behaves the same as BottomNavigationBar.
-        mainAxisAlignment: items.length <= 2
-            ? MainAxisAlignment.spaceEvenly
-            : MainAxisAlignment.spaceBetween,
-        children: [
-          for (final item in items)
-            TweenAnimationBuilder<double>(
-              tween: Tween(
-                end: items.indexOf(item) == currentIndex ? 1.0 : 0.0,
-              ),
-              curve: curve,
-              duration: duration,
-              builder: (context, t, _) {
-                final _selectedColor = item.selectedColor ??
-                    selectedItemColor ??
-                    theme.primaryColor;
-
-                final _unselectedColor = item.unselectedColor ??
-                    unselectedItemColor ??
-                    theme.iconTheme.color;
-
-                return Material(
-                  color: Color.lerp(
-                      _selectedColor.withOpacity(0.0),
-                      _selectedColor.withOpacity(selectedColorOpacity ?? 0.1),
-                      t),
-                  shape: itemShape,
-                  child: InkWell(
-                    onTap: () => onTap?.call(items.indexOf(item)),
-                    customBorder: itemShape,
-                    focusColor: _selectedColor.withOpacity(0.1),
-                    highlightColor: _selectedColor.withOpacity(0.1),
-                    splashColor: _selectedColor.withOpacity(0.1),
-                    hoverColor: _selectedColor.withOpacity(0.1),
-                    child: Padding(
-                      padding: itemPadding -
-                          (Directionality.of(context) == TextDirection.ltr
-                              ? EdgeInsets.only(right: itemPadding.right * t)
-                              : EdgeInsets.only(left: itemPadding.left * t)),
-                      child: Row(
-                        children: [
-                          IconTheme(
-                            data: IconThemeData(
-                              color: Color.lerp(
-                                  _unselectedColor, _selectedColor, t),
-                              size: 24,
+    return ColoredBox(
+      color: backgroundColor ?? theme.bottomAppBarColor,
```

Comment:
```suggestion
      color: backgroundColor ?? Colors.transparent
```

While theme.bottomAppBarColor is probably the right choice, it is a breaking change. So let's default to `Colors.transparent` so we don't have to do a major version release

Reply:
`Fixed`